kernel/arch/i386/CMakeLists.txt

```diff
@@ -1,6 +1,34 @@
 # SPDX-License-Identifier: BSD-2-Clause
 cmake_minimum_required(VERSION 3.2)
 
+# GNUEFI
```

Comment:
As discussed via e-mail, there's no need to attempt linking gnu-efi to the kernel. We only need structs and defines. It's only necessary the kernel to see the efiapi.h include file, as I mentioned in the email.